src/main/features/claim/form/models/interestTotal.ts

```diff
@@ -0,0 +1,49 @@
+import { IsDefined, MaxLength, Min } from 'class-validator'
+import { CompletableTask } from 'models/task'
+import { Fractions } from 'forms/validation/validators/fractions'
+import { ValidationErrors as CommonValidationErrors } from 'app/forms/validation/validationErrors'
+import { IsNotBlank } from 'forms/validation/validators/isBlank'
+import { toNumberOrUndefined } from 'common/utils/numericUtils'
+
+export class ValidationErrors {
+  static readonly REASON_REQUIRED: string = 'You must tell us how you calculated the amount'
+}
+
+export class InterestTotal implements CompletableTask {
+
+  @Min(0.01, { message: CommonValidationErrors.AMOUNT_NOT_VALID })
+  @Fractions(0, 2, { message: CommonValidationErrors.AMOUNT_INVALID_DECIMALS })
+  @IsDefined({ message: CommonValidationErrors.AMOUNT_REQUIRED })
```

Comment:
I think if you moved up the `IsDefined` up one you might not need to change the `Fractions` validator to accept `''`